### DISCUSSION TITLE
Improve reading experience on mobile

### DISCUSSION THREAD

**Discussion by aviggiano** [solodit/feedback#14] [New Feature Requests]
Like many others, I've been using solodit to learn from other auditors on my phone. Overall the reading experience is good, but there's one thing bothering me: the fact that report links open in a new page. This makes it difficult to navigate between multiple reports. 

For example, suppose I'm studying lending protocols. I want to review every single report available on solodit. The issue is that on my phone I'd need to click a link, go to a new page, read the report, close the page, go back to the previous page, and repeat this over and over again.

If links opened on the same page, this whole process could be much faster, not only because I wouldn't need to close new tabs every time, but also because my browser's phone would improve the performance of hitting the back button to go to the "previous page".

If others do not agree on this being the default, there could be a profile option to change the behavior per user.

**Comment by wang1252** [comment]
@aviggiano 

We understand your difficulties.
We will discuss what the best way is to overcome this, and handle this problem with high priority.
Thank you for your feedback and suggestion.
